nttt/tidyup.py

```diff
@@ -1,10 +1,8 @@
-from .cleanup_markdown import trim_spaces_on_specific_markdown
-import os
+import os, yaml
 from .constants import ArgumentKeyConstants, GeneralConstants
 from .utilities import add_missing_entries, find_files, find_replace, find_snippet, get_file, save_file
-import yaml
-import os.path
-
+from .cleanup_markdown import trim_spaces_on_specific_markdown
+from .tag_trimming import trim_tags
```

Comment:
In an effort to keep some order I have moved "external" imported modules such as `os` and `yaml` to the beginning and I have placed imports related to modules we have written after those. I have also erased the `os.path` import as we were already calling those module's functions through the `os.path.foo()` notation. If these were to be wrong changes they could be fixed within the `improvements` branch after merging the PR or we could open a new PR with this issue fixed.